IdentityCore/src/cache/token/MSIDCredentialCacheItem.m

```diff
@@ -218,16 +218,20 @@ - (BOOL)matchesTarget:(NSString *)target comparisonOptions:(MSIDComparisonOption
         return YES;
     }
     
+    if(comparisonOptions == MSIDExactStringMatch)
```

Comment:
seems weird that this option is not part of the switch.

Reply:
I am trying to avoid the perf hit of computing inputSet/tokenSet which are unused if `comparisonOptions == MSIDExactStringMatch`. But I don't care too deeply, let me know whether you prefer perf over readability